rkt/prepare.go

```diff
@@ -107,7 +107,9 @@ func runPrepare(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
-	if len(flagPodManifest) > 0 && (len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || flagStoreOnly || flagNoStore) {
+	if len(flagPodManifest) > 0 && (len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || flagStoreOnly || flagNoStore ||
+		(*appsVolume)(&rktApps).String() != "" || (*appMount)(&rktApps).String() != "" || (*appExec)(&rktApps).String() != "" ||
```

Comment:
seems I was a bit lost in types; I was thinking we could test the length of `rktApps`, e.g. by using `rktApps.Count()`, but that would only work for the truly positional flags like exec, etc, which mutate apps in there. But `appsVolume` and `appMount` (which should really be `appsMount`, AFAICT) are set on the `rktApps` structure. So we could still use `len(rktApps.Volumes)` and `len(rktApps.Mounts)` - but you might argue that's a little too much of an indirection relying on the flag parsing rather than presence of the flags. Thoughts?